src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -165,7 +165,13 @@ def get(self, request, organization):
         context = serialize(results, request.user, serializer())
 
         # HACK: remove auto resolved entries
-        if query_kwargs.get('status') == GroupStatus.UNRESOLVED:
+        # TODO: We should try to integrate this into the search backend, since
+        # this can cause us to arbitrarily return fewer results than requested.
```

Comment:
Could this also impact pagination?

Reply:
Shouldn't do, this filtering happens after we get the cursor, so paging should work. I haven't changed this behaviour in this diff, just adapted it to work with the new style of search params.